docs/docs/055-platforms/tf-aws.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Terraform/AWS
+id: tf-aws
+sidebar_label: tf-aws
+description: Terraform/AWS platform
+keywords: [Wing reference, Wing language, language, Wing language spec, Wing programming language, cli, terraform, aws, tf-aws, tfaws, amazon web services, platform]
+---
+
+The `tf-gcp` [platform](../02-concepts/03-platforms.md) compiles your program for Terraform and run on AWS.
+
+## Usage
+
+```sh
+$ wing compile --platform tf-aws [entrypoint]
+```
+
+## Parameters
+
+The `tf-aws` platform supports the following parameters (in `wing.toml`):
+
+* `vpc` - Determine whether to create a new VPC or use an existing one. Allowed values: `"new"` or `"existing"`.
+* `private_subnet_ids` (array of strings) - If using an existing VPC, provide the private subnet IDs.
+* `public_subnet_ids` (array of strings) - If using an existing VPC, provide the public subnet IDs.
+* `vpc_api_gateway` (boolean) - Whether Api gateways should be deployed in a VPC.
+* `vpc_lambda` (boolean) - Whether Lambda functions should be deployed in a VPC.
```

Comment:
I wonder how hard it would be to generate this list from the Json schema, since schemas support "descriptions". Totally out of scope here, but interesting idea

Reply:
Yes, the thought definitely went through my mind.